data/json/furniture_and_terrain/furniture-flora.json

```
@@ -60,7 +60,7 @@
   {
     "type": "furniture",
     "id": "f_maianthemum_stellatum",
-    "name": "starry false solomon's seal",
+    "name": "starry false Solomon's seal",
```

```suggestion
    "name": "Starry false Solomon's seal",
```

It appears that most if not all things are capitalized at the start.

---

Items aren't capitalized in their ``"name"`` unless they're a proper noun